host_applications/linux/apps/raspicam/RaspiVid.c

```diff
@@ -2401,6 +2400,10 @@ int main(int argc, const char **argv)
    if (state.timeout == -1)
       state.timeout = 5000;
 
+   // Setup for sensor specific parameters, only set W/H settings if zero on entry
```

Comment:
W/H settings?

Reply:
Wrong-Headed?

Reply:
Width/Height

Didn;t want to write too much..

Reply:
Ah, I'd assumed it was H/W with the letters reversed.
Surely the comment is applicable to the function, not the function call.

Reply:
Yeah, probably. I wanted to give some idea that the function wouldn't override any set default values since I had to look it up just now to confirm.